pandas/tests/io/test_sql.py

```diff
@@ -2698,6 +2700,20 @@ def test_drop_table(conn, request):
         assert not insp.has_table("temp_frame")
 
 
+@pytest.mark.parametrize("conn", sqlalchemy_connectable + ["sqlite_buildin"])
+def test_delete_rows_success(conn, test_frame1, request):
+    table_name = "temp_frame"
+    conn = request.getfixturevalue(conn)
+    pandasSQL = pandasSQL_builder(conn)
+    with pandasSQL.run_transaction():
+        assert (
+            pandasSQL.to_sql(test_frame1, table_name, if_exists="replace")
+            == test_frame1.shape[0]
+        )
+        assert pandasSQL.delete_rows(table_name) is None
```

Comment:
Can you add an assertion that the table does not actually get dropped?

Reply:
For sure, good catch !